src/commands/dao_cmds/install.js

```diff
@@ -182,15 +184,21 @@ exports.task = async ({
           return Promise.all(
             permissions.map(params => {
               const getTransactionPath = async wrapper => {
-                wrapper.getACLTransactionPath('createPermission', params)
+                return wrapper.getACLTransactionPath('createPermission', params)
               }
 
-              return execTask(dao, getTransactionPath, {
-                reporter,
-                apm: apmOptions,
-                web3,
-                wsProvider,
-              })
+              return (
+                execTask(dao, getTransactionPath, {
+                  reporter,
+                  apm: apmOptions,
+                  web3,
+                  wsProvider,
+                  silent,
+                  debug,
+                })
+                  // execTask returns a TaskList not a promise
+                  .then(tasks => tasks.run())
```

Comment:
The problem was `execTask` returns a `Listr.TaskList` not a promise, so we need to do `.run()` on the tasklist.